core/src/test/scala/org/apache/spark/CheckpointSuite.scala

```diff
@@ -584,7 +584,7 @@ object CheckpointSuite {
   }
 }
 
-class CheckpointCompressionSuite extends SparkFunSuite with LocalSparkContext {
+class CheckpointStorageSuite extends SparkFunSuite with LocalSparkContext {
```

Comment:
Could you mention this change in the PR description?

Reply:
ok.